### PR TITLE
Add Dmitrii Kilishek's profile

### PR DESCRIPTION
**Pull request by mentalclear**
## Changes proposed

Add Dmitrii Kilishek's profile to the project using dmitrii-kilishek.json file.

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers
For me cypress tests were failing even before I touched the code... so most likely I have nothing to do with them. I could run the code and see my profile with no issues nontheless. 

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1096"></a>

### CODE REVIEW

File: public/data/dmitrii-kilishek.json
Comment (KendallDoesCoding) [inline public/data/dmitrii-kilishek.json:23]:
```suggestion
      "name": "Visit my personal Blog",
```

typo

**Review by KendallDoesCoding — CHANGES_REQUESTED** [review]

**Review by schmelto — APPROVED** [review]

**Comment by schmelto** [comment]
LGTM Lets wait for another maintainer to approve 🚀

File: public/data/dmitrii-kilishek.json
Comment (KendallDoesCoding) [inline public/data/dmitrii-kilishek.json:4]:
```suggestion
  "bio": "Hey dear visitor! I'm Dmitrii. A US based Software Tester, Lifelong Learner, Former Sysadmin, In IT Since 1994, From: 🇷🇺 with ❤️",
```

Comment (KendallDoesCoding) [reply public/data/dmitrii-kilishek.json:4]: It can also be made United States or U.S if you want, but for now, it can remain as it is, but just capitalize it.

**Review by KendallDoesCoding — CHANGES_REQUESTED** [review]

File: public/data/dmitrii-kilishek.json
Comment (KendallDoesCoding) [inline public/data/dmitrii-kilishek.json:4]:
```suggestion
  "bio": "Hey dear visitor! I'm Dmitrii. A 🇺🇸 based Software Tester, Lifelong Learner, Former Sysadmin. I've been in IT since 1994, From: 🇷🇺 with ❤️",
```

Update bio.

**Review by KendallDoesCoding — CHANGES_REQUESTED** [review]

File: public/data/dmitrii-kilishek.json
Comment (mentalclear) [reply public/data/dmitrii-kilishek.json:4]: If you think that it makes more sense like this... why not.

**Review by kabir0x17 — CHANGES_REQUESTED** [review]
Incorrect spelling

File: public/data/dmitrii-kilishek.json
Comment (mentalclear) [reply public/data/dmitrii-kilishek.json]: All suggestions are addressed in the last commit. See: https://github.com/EddieHubCommunity/LinkFree/pull/1096/commits/bf14bf9f6f8881461c9f4d1b828791e7a86723ad

File: public/data/dmitrii-kilishek.json
Comment (mentalclear) [reply public/data/dmitrii-kilishek.json]: All suggestions are addressed in the last commit. See: https://github.com/EddieHubCommunity/LinkFree/pull/1096/commits/bf14bf9f6f8881461c9f4d1b828791e7a86723ad

**Review by KendallDoesCoding — APPROVED** [review]
Looks good to me!

**Pull request merged**
